### ISSUE TITLE
Publish Source and Docs

### ISSUE THREAD

**Issue by darkfrog26** [portable-scala/sbt-platform-deps#4]
In IntelliJ if you check to include SBT sources it will fail to import a project with the following:

```
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
[warn]     ::              FAILED DOWNLOADS            ::
[warn]     :: ^ see resolution messages for details  ^ ::
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
[warn]     :: org.portable-scala#sbt-platform-deps;1.0.0-M2!sbt-platform-deps.jar(doc)
[warn]     :: org.portable-scala#sbt-platform-deps;1.0.0-M2!sbt-platform-deps.jar(src)
[warn]     ::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by sjrd** [comment]
Could it be related to https://github.com/sbt/sbt/issues/3784?

**Comment by darkfrog26** [comment]
I'm using SBT 1.1.0, and if I'm reading that correctly, it should be fixed in that release, right?

**Comment by sjrd** [comment]
The changes in sbt 1.x were intentional. But it made some src and doc artifacts unresolvable (or resolved in slightly different ways) by older *consumers*. If IntelliJ resolves those things on its own, it might resolve them the wrong way (which was right before, but not anymore). What they *fixed* in sbt is that sbt 0.13.17 will now correctly resolve the artifacts published by sbt 1.x. 1.x was not changed to fix the bug.

**Comment by sjrd** [comment]
The sources and docs are definitely published on Bintray, as can be seen [here](https://dl.bintray.com/portable-scala/sbt-plugins/org.portable-scala/sbt-platform-deps/scala_2.12/sbt_1.0/1.0.0-M2/). Moreover, `updateClassifiers` in sbt can find them. So I'm going to close this. There might be something funny going on with IntelliJ's way of resolving sources and docs artifacts.
